t/ams/cd.xml

```diff
@@ -205,15 +221,15 @@
                 </XMCell>
                 <XMCell/>
                 <XMCell align="center">
-                  <XMTok fontsize="160%" name="uparrow" role="ARROW">↑</XMTok>
+                  <XMTok fontsize="160%" name="||" role="VERTBAR">∥</XMTok>
                 </XMCell>
                 <XMCell/>
                 <XMCell align="center">
-                  <XMTok fontsize="160%" name="uparrow" role="ARROW">↑</XMTok>
+                  <XMTok fontsize="160%" name="||" role="VERTBAR">∥</XMTok>
```

Comment:
The `name="||"` got me curious, so I checked the DefMath, which indeed uses that name for `\|`. 

```perl
DefMathI('\|',        undef, "\x{2225}", role => 'VERTBAR',  name    => '||');
```

In the TeXbook, I see its definition marked with "vertical line" in Appendix E, p. 420:
```
\def\|{\leavevmode\hbox{\tt\char‘\|}} % vertical line
```

Would `vertical-lines` be a better name in this case?

Reply:
It *might* be, but `name` is a bit of an orphan: it's somewhere between the content, reversion and meaning; it probably shouldn't even exist at all, but it does get used for various checks.
At some point in the not too distant future, a scan of all uses needs to be analyzed and figure out where to go with it. Perhaps it should end up being part of intent infrastructure?  So: Hold onto that thought!